.env-example

```diff
@@ -1,33 +1,79 @@
-BROWSER_TRUSTED="false"
-DISCORD_NOTIFY_GROUP="@here"
-DISCORD_WEB_HOOK="https://discordapp.com/api/webhooks/23123123123/5dfsdfh3h2j5hdfhsdfsdhj55jf"
-EMAIL_USERNAME="youremail@gmail.com"
-EMAIL_PASSWORD="secretpassword"
-HEADLESS="true"
-IN_STOCK_WAIT_TIME="30"
-LOG_LEVEL="info"
+// All configuration variables are optional.  Please read https://github.com/jef/nvidia-snatcher#customization for help on customizing this file.
```

Comment:
I don't mind this top comment, but do you mind reverting the rest? I feel like it'll be hectic keeping up with doc here and `README` and everywhere else.

Reply:
All set